code/game/objects/items/stacks/sheets/hot_ice.dm

```diff
@@ -11,3 +11,10 @@
 /obj/item/stack/sheet/hot_ice/suicide_act(mob/living/carbon/user)
 	user.visible_message("<span class='suicide'>[user] begins licking \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
 	return FIRELOSS//dont you kids know that stuff is toxic?
+
+/obj/item/stack/sheet/hot_ice/fire_act(exposed_temperature, exposed_volume)
```

Comment:
This should be a component on the material.

Reply:
it was, it wasn't working, that's why i put it on the sheet themself

Reply:
Then you should fix the component rather than doing a work-around and putting it on the sheet itself.

Reply:
i tried, and i can't manage to make it work. I dont understand why the fire_act() proc is not igniting the hot ice when there is a fire

Reply:
reverting to this isn't a good alternative either way

Reply:
eh i know, but what i'm supposed to do? this is the best way i can fix this without removing the datum mat

Reply:
look into how to fix the component

Reply:
ok can't fix it, the proc is broken. Seems to be an hotspot issue. I'll leave the burn if on fire stuff on the sheets for now

Reply:
I really can't believe this is something unfixable and making a bugfix thats basically "I did it the wrong way instead" is not excusable because you couldn't debug the issue.

Reply:
Maybe I didn't explain it better.
I went back to check on the issue, I tried to make the proc work, but it is not getting called. I asked a friend of mine and he said that the proc seems to be broken, we did some tests and it turns out that even the thermite component fire_react doesn't work and the issue cold be the Hotspots. Now yes I may not have looked into it for starters and went the easy way, but now I searched extensively for a result that is way out my skill. I tried and I can't fix it.

Reply:
Even then this only fixes it for sheets which is not a fix its a band-aid.

Reply:
is a band-aid yes but who cares? you can only have the sheets in the game

Reply:
Because now if someone decides to add hot ice sculpting for god knows what reason the code is still broken which is a shitty practice? I spent the last minute investigating why this happens and obj/fire_act() does not call parent which is literaly the only issue.

Reply:
And what's the best way to fix this?

Reply:
add . = ..() in obj/fire_act()